modules/custom/az_event/az_event_trellis/config/install/migrate_plus.migration.az_trellis_events.yml

```diff
@@ -90,4 +103,6 @@ destination:
   plugin: 'entity:node'
   default_bundle: az_event
   validate: false
-migration_dependencies: null
+migration_dependencies:
+  optional:
```

Comment:
Is this really an optional dependency?

Reply:
I guess I'm not sure. Given that this migration is run on a per-node basis rather than in its entirety, I wasn't sure if there'd be an issue with marking it as actually required. It seemed to me that there'd be cases where entities being missing from one migration didn't matter as long as it wasn't one the node currently being imported depended on.

Reply:
Sounds good.